### PR TITLE
Add Back button to particular UIs

### LINKED ISSUE #74: Add Back button to UIs
**Describe the bug**
There isn't any back button in the UIs which is a major UX issue. UIs that require a back button is 'ManageMentees', 'MentorApplication', 'RequestMentors' and 'MentorConfirmation'.

**Expected behavior**
There must be a back button on the top of the screen to navigate back to the previous UI.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Chromium
 - Version: 67.0.3369.0

### PR DESCRIPTION
**Pull request by miluckshan-j**
## Purpose
The purpose of this PR is to fix #74

## Goals
Since there wasn't a back button before, there was a hassle in navigating to previous page. By adding a button to navigate back, the UX can be improved.

## Approach
By using the useHistory() hook, we can navigate to previous pages on a press of a button.

### Manage Mentees
ManageMentees

### Mentor Application
MentorApplication

### Mentor Confirmation
MentorConfimation

### Request Mentor
RequestMentor

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

## Test environment
Windows 10
Chromium Version 67.0.3369.0

## Learning
https://reactrouter.com/web/api/Hooks/usehistory

### CODE REVIEW

File: src/scenes/Home/components/ActivePrograms/index.tsx [hunk 1]
```diff
@@ -39,70 +39,76 @@ function ActivePrograms() {
...
+                          rel={'noreferrer'}
+                          href={program.landingPageUrl}
+                        >
+                          {program.title}
+                        </a>
+                      </Title>
+                    </Col>
+                    <Col span={11} className={styles.programActionButton}>
+                      <Button
+                        hidden={!isUserAdmin}
```
Comment (anjula-sack) [inline src/scenes/Home/components/ActivePrograms/index.tsx]: In here also let's use conditional rendering

File: src/scenes/Home/scenes/MentorApplication/index.tsx [hunk 2]
```diff
@@ -96,6 +110,7 @@ function MentorApplication() {
             <Row>
               <Col md={2} />
               <Col md={12}>
+                {/* MJ CHECK */}
```
Comment (anjula-sack) [inline src/scenes/Home/scenes/MentorApplication/index.tsx]: What is this?

**Review by anjula-sack — COMMENTED** [review]
Can you add some screenshots please?

File: src/scenes/Home/components/ActivePrograms/index.tsx (hunk 1 above)
Comment (miluckshan-j) [reply src/scenes/Home/components/ActivePrograms/index.tsx]: Sure.

File: src/scenes/Home/scenes/MentorApplication/index.tsx (hunk 2 above)
Comment (miluckshan-j) [reply src/scenes/Home/scenes/MentorApplication/index.tsx]: Was checking some files. Forgot to remove the comments. Apologies.

**Review by anjula-sack — APPROVED** [review]
Thank you! @miluckshan-j

**Pull request merged**